src/js/tinymce_plugins/schematags.js

```diff
@@ -326,7 +325,7 @@
 					} else {
 						highLevelAtts += currAttString;
 					}
```

Comment:
The indentation of this chunk of code from which the `if` condition was removed should probably be made to match the surrounding code.